src/client/src/containers/Header/index.tsx

```diff
@@ -18,6 +18,7 @@ const Header = () => {
   return (
     <div className={styles.header}>
       <div className={styles.headerTitle}>Web Template Studio</div>
+      <div className={styles.headerTitleSmall}>WTS</div>
```

Comment:
Would it be possible to call it WebTS instead? Just because there may be confusion with WTS.

Reply:
Also, shouldn't these be internationalized?

Reply:
Just changed it to WebTS :)

Reply:
@jcoc611-microsoft I don't think it's necessary to have these internationalized just because it should be the same no matter what language it changes to